pkg/nad/masterplugin.go

```diff
@@ -393,3 +393,126 @@ func (plugin *MasterIPVlanPlugin) GetMasterPluginConfig() (*MasterPlugin, error)
 
 	return plugin.masterPlugin, nil
 }
+
+// MasterBondPlugin provides struct for MasterPlugin to create a NAD for bonded interfaces.
+type MasterBondPlugin struct {
+	masterPlugin *MasterPlugin
+	errorMsg     string
+}
+
+// NewMasterBondPlugin creates new instance of MasterBondPlugin.
+func NewMasterBondPlugin(name, mode string) *MasterBondPlugin {
+	validModes := map[string]bool{
+		"balance-rr":    true,
+		"active-backup": true,
+		"balance-xor":   true,
+	}
+
+	glog.V(100).Infof(
+		"Initializing new MasterBondPlugin structure %s", name)
```

Comment:
Please add mode to the log:
glog.V(100).Infof("Initializing new MasterBondPlugin structure %s with mode %s", name, mode)